tests/testthat/test-user_utils.R

```diff
@@ -46,15 +46,16 @@ test_that("convert_blanks_to_na.list Test 4: `convert_blanks_to_na.list` produce
 })
 
 #  Test 5: convert_na_to_blanks Test 5----
```

Comment:
@bms63 this `testthat` file is incorrectly formatted, I can go in and fix it then we can merge after?